cpp/include/raft/neighbors/detail/cagra_hnswlib.hpp

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
I think raft/neighbors/detail/hnsw.hpp would be more appropriate. By this point we have removed cagra completely from the story.